data/schemas/keyboard.jsonschema

```diff
@@ -431,6 +457,27 @@
                     "type": "string",
                     "enum": ["eeprom", "left", "matrix_grid", "pin", "right"]
                 },
+                "pointing_device": {
+                    "enabled": {"type": "boolean"},
+                    "right": {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "type": "object",
+                        "properties": {
+                            "side": {
+                                "type": "string",
+                                "enum": ["left", "right", "combined"]
+                            },
+                            "invert_x": {"type": "boolean"},
+                            "invert_y": {"type": "boolean"},
+                            "rotation": {
+                                "type": "integer",
+                                "minimum": 0,
+                                "enum": [0, 90, 180, 270]
+                            }
+                        }
+                    }
+                },
```

Comment:
Is the duplication intended?

Reply:
Oh I completely missed the `right` part here! Re-use is not easily possible so far I guess.